package.json

```diff
@@ -61,7 +61,8 @@
     "typescript-eslint": "^7.5.0",
     "unplugin-swc": "1.4.4",
     "vitest": "1.3.1",
-    "wsdl-tsclient": "1.3.1"
+    "wsdl-tsclient": "1.3.1",
+    "catch-unknown": "2.0.0"
```

Comment:
Nice, catch-unknown, finally (pun intended)! But, can you move this to normal dependencies instead of dev-seps?